commercial/app/views/books/book.scala.html

```diff
@@ -30,7 +30,7 @@ <h4 class="lineitem__title" itemprop="name">@book.title</h4>
             </p>
             </a>
             <a href="@clickMacro@book.buyUrl" class="lineitem__cta button button--primary button--small" data-link-name="merchandising-bookshop-v@{version}_@{date}-superHigh-visit-shop">
-                Buy now<i class="i i-arrow-white-right i-right"></i>
+                Buy now@fragments.inlineSvg("arrow-white-right", "icon", List("i-right"))
```

Comment:
plan was to use `inline-` prefix for all these, until we've removed everything from css and then revert all inlined icons (i.e. by then just 'icons') to the `i-` prefix. think it would be useful if it's simple, but it's not worth breaking backs over it if it's not straightforward.